Importing process records fail with PostgreSQL

On starting zeeqs with postgres db ([docker-compose file](https://pastebin.pl/view/e0cb9820)) the log file is filled up with errors -
`Caused by: org.postgresql.util.PSQLException: ERROR: invalid byte sequence for encoding "UTF8": 0x00`
(complete log file is shared [here](https://pastebin.pl/view/609467ce)). 

On submitting the process definition to zeebe we sometimes get null values in the api response instead of the process instance information. The process definition retured is also null. Linked are the [screenshots](https://files.slack.com/files-pri/T6VHNLZRR-F03LTPM2C06/image.png) of the [graphql requests](https://files.slack.com/files-pri/T6VHNLZRR-F03M2Q80FL4/image.png) made through postman client. 

On a slack discussion [here](https://camunda-platform.slack.com/archives/C6WGNHV2A/p1656333527514059?thread_ts=1655899938.176649&cid=C6WGNHV2A), the issue was reproduced when running zeeqs with postgres.

--- a/data/src/main/kotlin/io/zeebe/zeeqs/data/entity/Process.kt
+++ b/data/src/main/kotlin/io/zeebe/zeeqs/data/entity/Process.kt
@@ -12,4 +12,4 @@ data class Process(
         @Lob val bpmnXML: String,
         val deployTime: Long,
         val resourceName: String,
-        val checksum: String)
+        @Lob val checksum: String)

